projeto-semana5-whats4/src/components/Usuario/Usuario.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './Usuario.css';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+
+class Usuario extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      arrayUserMsg : [],
+    };
```

Comment:
É uma prática ruim utilizar states que não estão no constructor. Então aqui deveria ser:
this.state = {
  arrayUserMsg : [],
  nomeUser: '',
  textoMensagem: ''
};